src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -312,7 +312,7 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
                 var settings = serviceProvider.GetRequiredService<ShellSettings>();
                 var environment = serviceProvider.GetRequiredService<IHostEnvironment>();
 
-                var cookieName = "orchantiforgery_" + HttpUtility.UrlEncode(settings.Name + environment.ContentRootPath);
+                var cookieName = "orchantiforgery_" + HttpUtility.UrlEncode(settings.Name + environment.ContentRootPath.Replace("(", "").Replace(")", ""));
```

Comment:
I think the issue is with the encoding logic, `UrlEncode` is apparently not adapted. Base64 or Uri.EscapeDataString (something close).

Also we can use a Guid.ToString("n") instead of the `ContentRootPath`. 
Details: https://github.com/OrchardCMS/OrchardCore/pull/5803